interview-questions.md

```diff
@@ -4,41 +4,58 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: You would use the super() method in a class that also uses constructor()
 
-  Researched answer:
+  Researched answer: "By calling the super() method in the constructor method, we call the parent's constructor method and gets access to the parent's properties and methods" 
+  Source - https://www.w3schools.com/jsref/jsref_class_super.asp
 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
+  Your answer: It is the virtual "memory" of a UI. 
 
-  Researched answer:
+  Researched answer: "The virtual DOM (VDOM) is a programming concept where an ideal, or “virtual”, representation of a UI is kept in memory and synced with the “real” DOM by a library such as ReactDOM."
+  Source - https://reactjs.org/docs/faq-internals.html
 
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
-  Your answer:
+  Your answer: After creating the class and extending it to Component you would have to code in the render() and return() methods to house "Hello World" in between some sort of HTML tags.
```

Comment:
Pretty close! You'd also need to have an import and an export. Otherwise good answer!